app/templates/codelists-management/codelist.hbs

```diff
@@ -0,0 +1,110 @@
+{{page-title (t "codelist.details")}}
+
+<BreadcrumbsItem as |linkClass|>
+  <LinkTo
+    @route="codelists-management.codelist"
+    @model={{@model.codelist.id}}
+    class={{linkClass}}
+  >
+    {{@model.codelist.readableLabel}}
+  </LinkTo>
+</BreadcrumbsItem>
+
+<div class="au-o-grid au-o-grid--flush au-o-grid--fixed">
+  <div
+    class="au-o-grid__item {{
+    if
+      this.showSidebar
+      "au-u-3-5@medium au-u-visible-from@medium"
+      "au-u-1-1@medium"
+    }}"
+  >
+    <div class="au-c-body-container au-c-body-container--scroll">
+      <AuToolbar @size="large">
+        <AuToolbarGroup>
+          <AuHeading @skin="2">
+            {{t "codelist.details"}}
+          </AuHeading>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuToolbar @size="medium">
+        <AuToolbarGroup class="au-c-toolbar__group--row">
+          <ul class="au-o-grid">
+            <li class="au-o-grid__item au-u-1-2 au-u-1-6@medium">
+              <AuLabel>
+                {{t "codelist.attr.label"}}
+              </AuLabel>
+              <p>
+                {{@model.codelist.readableLabel}}
+              </p>
+            </li>
+            <li class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
+              <AuLabel>
+                {{t "codelist.attr.values"}}
+              </AuLabel>
+              <p>
+                <ul>
```

Comment:
This looks a bit weird without any list styles. Not sure if a class exists for this in Appuniversum though.

Reply:
Not sure whan appuniversum this is running but https://appuniversum.github.io/ember-appuniversum/docs/components/au-table#regular-table might be nice as well?

Reply:
We'll for sure, but i'll follow your nice suggestions so that in the meantime it doesn't look so bad ;)